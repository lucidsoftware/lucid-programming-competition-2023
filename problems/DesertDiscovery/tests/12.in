1
SpmM7cq3fCbXtp1w95U7CJ3tGjofrjAQci
8
AtJu8rPckAqVibqJ6bwu4Yp5N44JzYF7k9
AtpVjSM5IGHQ7VUDgYpum6PrC3Z6cuDdHD
AtWFBb1ngS3L7JfCfK7oaNSwnAfZ9rA0S5
AtEtQpfIfk2SZfocr02K5DtfL4rPtyCuu5
AtJs6IsPT3S2cwvVZdufYdCCpsvMh7O5Gs
AtaUUWJjyBV05NC3F9NsHVNLpKiYRAKpdD
AthRJutssfJsJFEMdscnkj0rFOrVnORUgH
At2qBpG6sOXZxuztNq8taIyxtNUAIYGn56
1
8
AtCPaSNESqTCyOPFETQms0PFuOU3VE1wH6
AtaUUWJjyBV05NC3F9NsHVNLpKiYRAKpdD
AtJs6IsPT3S2cwvVZdufYdCCpsvMh7O5Gs
AtnOejxCJNV0kD8lR3dqRwDNzqbe02N7ly
AtpVjSM5IGHQ7VUDgYpum6PrC3Z6cuDdHD
At2qBpG6sOXZxuztNq8taIyxtNUAIYGn56
AthRJutssfJsJFEMdscnkj0rFOrVnORUgH
AtEtQpfIfk2SZfocr02K5DtfL4rPtyCuu5
