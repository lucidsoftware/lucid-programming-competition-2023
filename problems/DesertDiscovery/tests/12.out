SpmM7cq3fCbXtp1w95U7CJ3tGjofrjAQci
